Sources/Util/Util.swift

```diff
@@ -22,6 +22,17 @@ public func isAnImage(_ file: String) -> Bool {
            file.hasSuffix("png")
 }
 
+public func getRange(of rawRange: String, length contentLength: Int) -> Range<Int> {
+  let splitRange = rawRange.components(separatedBy: "-")
+
+  let rangeEnd = Int(splitRange.last!) ?? contentLength - 1
+
+  let rangeStart = Int(splitRange.first!) ?? contentLength - rangeEnd
```

Comment:
Since I need data to be persisted across multiple requests, I created static variables within the controllers that need them. I'm using the `PersistentDataController` protocol to group these Controllers.

This works in implementation, but my tests are now order dependent unless I clear the contents to isolate each unit test. Is this a fair trade-off?